tsconfig.json

```diff
@@ -16,6 +16,7 @@
       "dom",
       "esnext"
     ],
+    "skipLibCheck": true,
```

Comment:
[Updated master](https://github.com/HospitalRun/components/blob/master/tsconfig.json#L21) currently has this added. Would you fetch and merge, and pls ensure this file is updated on your end?